templates/superset_config.py

```diff
@@ -101,12 +101,10 @@ class CeleryConfig(object):
 TIME_ROTATE_LOG_LEVEL = "DEBUG"
 ENABLE_TIME_ROTATE = True
 
-# html santization
-HTML_SANITIZATION = os.getenv("HTML_SANITIZATION")
-sanitization_extention = os.getenv("HTML_SANITIZATION_SCHEMA_EXTENSIONS")
-
-if HTML_SANITIZATION and sanitization_extention:
-    HTML_SANITIZATION_SCHEMA_EXTENSIONS = sanitization_extention
+# html sanitization
+HTML_SANITIZATION = os.getenv("HTML_SANITIZATION").lower() != "false"
```

Comment:
Invalid values will default to `True` instead of `False`. If you are happy with this, then OK.

Reply:
Yes this would be the preference